store/rootmulti/store.go

```diff
@@ -361,7 +379,9 @@
```

Comment:
Why not protecting the other fields? Are the access to them safe?

Reply:
do you mean storesParams and keysByName, which depends on how we use but we mainly encounter stores concurrent write